### PR TITLE
Update _LANGUAGE_GUESS to include new languages

### PR DESCRIPTION
**Pull request by JoelNiemela**

### CODE REVIEW

File: submit.py [hunk 1]
```diff
@@ -81,7 +92,7 @@
     4: 'Waiting for run',
     _RUNNING_STATUS: 'Running',
     6: 'Judge Error',
-    7: 'Submission Error',
+    # 7: '<invalid value>',
```
Comment (tyilo) [inline submit.py:95]: Why was this removed?

Comment (JoelNiemela) [reply submit.py:95]: Submission Error is an old judgment status that can't occur anymore (similar to ID 15). I should have mentioned this when creating the PR.
